src/matchers/doc.rs

```diff
@@ -67,7 +67,12 @@ fn msooxml(buf: &[u8]) -> Option<DocType> {
     // skip to the second local file header
     // since some documents include a 520-byte extra field following the file
     // header, we need to scan for the next header
-    let mut start_offset = (u32::from_le_bytes(buf[18..22].try_into().unwrap()) + 49) as usize;
+    let mut start_offset = match u32::from_le_bytes(buf[18..22].try_into().unwrap()).checked_add(49)
```

Comment:
Here's a file (generated by fuzz tests) that replicates this:
[overflow.docx](https://github.com/bojand/infer/files/8854575/overflow.docx)